--- Python-2.7.13/Python/bltinmodule.c	2016-12-17 21:05:07.000000000 +0100
+++ Python-2.7.13-modified/Python/bltinmodule.c	2017-08-10 22:14:26.490843018 +0100
@@ -22,7 +22,7 @@
 #elif defined(__APPLE__)
 const char *Py_FileSystemDefaultEncoding = "utf-8";
 #else
-const char *Py_FileSystemDefaultEncoding = NULL; /* use default */
+const char *Py_FileSystemDefaultEncoding = "utf-8"; /* use default */
 #endif
 
 /* Forward */
