--- a/setup.py	2016-08-22 19:45:52.000000000 +0100
+++ b/setup.py	2017-09-16 01:35:36.675202502 +0100
@@ -1,11 +1,7 @@
 import glob
-from distutils.core import setup, Extension
-
-try:
-    from Cython.Build import cythonize
-except ImportError:
-    raise RuntimeError('Cython must be installed to build unqlite-python.')
-
+from setuptools import setup
+from setuptools.extension import Extension
+from setuptools.command.build_ext import build_ext
 
 python_source = 'unqlite.pyx'
 library_source = glob.glob('src/*.c')
@@ -45,5 +41,7 @@
         'Topic :: Database :: Database Engines/Servers',
         'Topic :: Software Development :: Embedded Systems',
         'Topic :: Software Development :: Libraries :: Python Modules'],
-    ext_modules=cythonize(unqlite_extension)
+    cmdclass = {'build_ext': build_ext},
+    ext_modules=[unqlite_extension]
 )
+
