--- a/Makefile	2019-03-29 13:35:22.677487223 +0100
+++ b/Makefile	2019-03-29 13:36:00.289935323 +0100
@@ -85,14 +85,14 @@
 # and pass it down to assembler.  In any case, we do not define AS or
 # ASFLAGS for this reason.
 
-CROSS_COMPILE=arm-linux-androideabi-
+CROSS_COMPILE=
 CC=$(CROSS_COMPILE)/usr/bin/ccache arm-linux-androideabi-gcc -DANDROID -fomit-frame-pointer -D__ANDROID_API__=21 -mandroid -I/home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/platforms/android-21/arch-arm -isysroot /home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/platforms/android-21/arch-arm -I/home/akinwale/Dev/Python/lbry-android/.buildozer/android/platform/build/build/python-installs/browser/include/python3.7   --sysroot=/home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/platforms/android-21/arch-arm -lm -L/home/akinwale/Dev/Python/lbry-android/.buildozer/android/platform/build/build/libs_collections/browser/armeabi-v7a -L/home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/sources/crystax/libs/armeabi-v7a -lcrystax
 CXX=$(CROSS_COMPILE)/usr/bin/ccache arm-linux-androideabi-g++ -DANDROID -fomit-frame-pointer -D__ANDROID_API__=21 -mandroid -I/home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/platforms/android-21/arch-arm -isysroot /home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/platforms/android-21/arch-arm -I/home/akinwale/Dev/Python/lbry-android/.buildozer/android/platform/build/build/python-installs/browser/include/python3.7
 CPPFLAGS=
 CFLAGS=-DANDROID -fomit-frame-pointer -D__ANDROID_API__=21 -mandroid -I/home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/platforms/android-21/arch-arm -isysroot /home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/platforms/android-21/arch-arm -I/home/akinwale/Dev/Python/lbry-android/.buildozer/android/platform/build/build/python-installs/browser/include/python3.7 -I/home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/sources/python/3.7/include/python/ -march=armv7-a -mfloat-abi=softfp -mfpu=vfp -mthumb   --sysroot=/home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/platforms/android-21/arch-arm -lm -L/home/akinwale/Dev/Python/lbry-android/.buildozer/android/platform/build/build/libs_collections/browser/armeabi-v7a -L/home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/sources/crystax/libs/armeabi-v7a -lcrystax
 CXXFLAGS=-DANDROID -fomit-frame-pointer -D__ANDROID_API__=21 -mandroid -I/home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/platforms/android-21/arch-arm -isysroot /home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/platforms/android-21/arch-arm -I/home/akinwale/Dev/Python/lbry-android/.buildozer/android/platform/build/build/python-installs/browser/include/python3.7 -I/home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/sources/python/3.7/include/python/ -march=armv7-a -mfloat-abi=softfp -mfpu=vfp -mthumb
 LDFLAGS=   --sysroot=/home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/platforms/android-21/arch-arm -lm -L/home/akinwale/Dev/Python/lbry-android/.buildozer/android/platform/build/build/libs_collections/browser/armeabi-v7a -L/home/akinwale/Dev/SDKs/crystax-ndk-10.3.2/sources/crystax/libs/armeabi-v7a -lcrystax
-EX_LIBS= 
+EX_LIBS=
 
 MAKEDEPEND=
 
